### ISSUE TITLE
Not working in Android with RN 0.33

### ISSUE THREAD

**Issue by tioback** [greatbsky/react-native-webview2#5]
After updating to RN 0.33, I started getting errors with `_onLoadingStart` and `onLoadingFinish` when `super` are called.

```
09-18 04:42:12.966  2103 27828 I ReactNativeJS: 'error while calling super onloadingstart', { [TypeError: undefined is not an object (evaluating 'babelHelpers.get(_class.prototype.__proto__||Object.getPrototypeOf(_class.prototype),'onLoadingStart',this).call')]
09-18 04:42:12.966  2103 27828 I ReactNativeJS:   line: 91823,
09-18 04:42:12.966  2103 27828 I ReactNativeJS:   column: 108,
09-18 04:42:12.966  2103 27828 I ReactNativeJS:   sourceURL: 'http://10.0.3.2:8081/index.android.bundle?platform=android&dev=true&hot=false&minify=false' }
09-18 04:42:13.005  2103 27828 I ReactNativeJS: 'Error while running super.onloadingFinish', { [TypeError: undefined is not an object (evaluating 'babelHelpers.get(_class.prototype.__proto__||Object.getPrototypeOf(_class.prototype),'onLoadingFinish',this).call')]
09-18 04:42:13.005  2103 27828 I ReactNativeJS:   line: 91837,
09-18 04:42:13.005  2103 27828 I ReactNativeJS:   column: 109,
09-18 04:42:13.005  2103 27828 I ReactNativeJS:   sourceURL: 'http://10.0.3.2:8081/index.android.bundle?platform=android&dev=true&hot=false&minify=false' }
```

Temporary solution was to surround both calls with try/catch blocks:

```
_onLoadingStart(e) {
    // console.log(`............................_onLoadingStart`);
    // console.log(e.nativeEvent);
    if (getHash(e.nativeEvent.url).indexOf(this.props.pageId) == -1) {

      try {
        super.onLoadingStart(e);
      } catch(e) {
        console.log("error while calling super onloadingstart", e);
      }
    }
}

_onLoadingFinish (e) {
    // console.log('...................onLoadingFinish current url:' + e.nativeEvent.url);
    let hash = getHash(e.nativeEvent.url);
    if (hash.indexOf(this.props.pageId) == 0) { // 非原始页，需要执行返回结果
        this.props.evalReturn && this.props.evalReturn(decodeURIComponent(hash)); //执行返回结果
    } else if (hash.indexOf(this.props.pageId) == -1) {
      try {
        super.onLoadingFinish(e);
      } catch (err) {
        console.log("Error while running super.onloadingFinish", err)
      }
        this.props.initJavaScript && this.props.initJavaScript();
    }
}
```

**Comment by yilianhuaixiao** [comment]
RN 0.32, same question

**Comment by tioback** [comment]
I don't know where those `super.onLoadingFinish` and `super.onLoadingStart` came from, because React Native's original WebView only has `onLoadStart` and `onLoadEnd`.

Does anybody know anything about this?

-- EDIT --
I tried changing the calls to `super.onLoadingStart` and `super.onLoadingFinish` to `super.onLoadStart` and `super.onLoadEnd` respectively, and it stopped crashing. However, the auto-height behavior was gone.

So, I abandoned it in favor of [this gist](https://gist.github.com/epeli/10c77c1710dd137a1335). Hope it helps anyone going through the same problems.

**Comment by greatbsky** [comment]
sorry for waiting, I will think about this this week, I am busy these days...

**Comment by greatbsky** [comment]
This issue has been fixed at version 1.0.2. now support reactnative 0.34.0

**Comment by greatbsky** [comment]
it's work well on android, so I close this issue, if you have question, tell me please
